armada_command/command_restart.py

```diff
@@ -29,11 +29,11 @@ def command_restart(args):
     if args.verbose:
         global verbose
         verbose = True
-    microservice_name = args.microservice_name or os.environ['MICROSERVICE_NAME']
-    if not microservice_name:
+    microservice_handler = args.microservice_handler or os.environ['MICROSERVICE_NAME']
+    if not microservice_handler:
         raise ValueError('No microservice name supplied.')
```

Comment:
I would also add "container id" to the exception message, as in the other file:
"No microservice name or container id supplied."